wandb/sdk/verify/verify.py

```diff
@@ -79,8 +79,6 @@ def check_logged_in(api: Api, host: str) -> bool:
                 "Could not get viewer with default API key. "
                 f"Please relogin using `WANDB_BASE_URL={host} wandb login --relogin` and try again"
             )
-
-    print_results(fail_string, False)
```

Comment:
Suspicious of this change, use
`git checkout master -- wandb/sdl/verify/verify.py`
to reset the file and push it

Reply:
Arg, i fixed this file and git stashed it briefly, no git pop :(